programs/cltester/main.cpp

```diff
@@ -1555,21 +1670,53 @@ static void run(const char* wasm, const std::vector<std::string>& args)
    backend(cb, "env", "_start");
 }
 
-const char usage[] = "usage: cltester [-h or --help] [-v or --verbose] file.wasm [args for wasm]\n";
+const char usage[] = "USAGE: cltester [OPTIONS] file.wasm [args for wasm]...\n";
+const char help[] = R"(
+OPTIONS:
+      -h, --help
+
+            Show this message
+
+      -v, --verbose
+
+            Show detailed logging
+
+      -s contract.wasm debug.wasm
+      --subst contract.wasm debug.wasm
```

Comment:
I wonder if external debug info works: https://yurydelendik.github.io/webassembly-dwarf/#external-DWARF.  Also, is it possible to use RelWithDebugInfo?

Reply:
Clang in wasm mode doesn't produce dwarf at `-O3` or `-Os`. I suspect it's because it invokes `wasm-opt`